Modix/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -48,6 +50,10 @@ public static IServiceCollection AddModixHttpClients(this IServiceCollection ser
         {
             services.AddHttpClient();
 
+            services.AddHttpClient(HttpClientNames.RetryOn5xxPolicy)
+                .AddPolicyHandler(HttpPolicyExtensions.HandleTransientHttpError()
```

Comment:
Is a "Transient Http Error" defined as a 5xx? Otherwise, I don't see how "RetryOn5xx" is an appropriate name for this policy.

Reply:
Network failures (as HttpRequestException)
HTTP 5XX status codes (server errors)
HTTP 408 status code (request timeout)

https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.pollyhttpclientbuilderextensions.addtransienthttperrorpolicy?view=dotnet-plat-ext-3.1#remarks

Reply:
I'd say call it `RetryOnTransientErrorPolicy` then. But feel free to merge it anyway.

Reply:
updated the name